README.md

```diff
@@ -28,22 +28,53 @@ We welcome contributions to amp.dev.
     $ nvm install --lts
     ```
 
+1. Install Python 2.7 and ensure pip is properly set up by adding the _pip user base binary directory_ to `$PATH`.
```

Comment:
Can you mark this step as optional or mention that this is only required if python and pip are not yet installed.

Reply:
Alright, done.